cliquet/resource.py

```diff
@@ -12,7 +12,7 @@
 from cliquet import Service
 from cliquet.collection import Collection
 from cliquet.errors import http_error, raise_invalid, ERRORS
-from cliquet.schema import ResourceSchema, PermissionsSchema
+from cliquet.schema import Schema, PermissionsSchema
```

Comment:
I was wondering that too... But maybe it's ok to just have ``cliquet.Schema`` and ``cliquet.Resource``.